web/packages/teleport/src/config.ts

```diff
@@ -203,6 +204,14 @@ const cfg = {
 
     downloadCenter: '/web/downloads',
 
+    // sso routes
+    /**
+     * ssoConnectorCreate is the dedicated page for creating a new auth connector.
+     */
+    ssoConnectorCreate: '/web/sso/new/:connectorType(github|oidc|saml)',
+    ssoConnectorEdit:
+      '/web/sso/edit/:connectorType(github|oidc|saml)/:connectorName?',
```

Comment:
suggestion: group these together in an object, makes it easier to find/update them later. 
```
ssoConnector : {
  Create: '/web/sso/new/:connectorType(github|oidc|saml)',
  Edit: '/web/sso/edit/:connectorType(github|oidc|saml)/:connectorName?'
}
```